PackageInfo.g

```diff
@@ -9,8 +9,8 @@ PackageName := "Cryst",
 
 Subtitle := "Computing with crystallographic groups",
 
-Version := "4.1.23",
-Date := "10/12/2019", # dd/mm/yyyy format
+Version := "4.1.24",
+Date := "24/05/2021", # dd/mm/yyyy format
```

Comment:
Please do not modify `PackageInfo.g` - it will be updated when the new release is published (maybe a different date, a different version, etc.). It also will require updating the changelog.

Reply:
Done.